3rdparty/python/requirements.txt

```diff
@@ -25,5 +25,5 @@ scandir==1.2
 setproctitle==1.1.10
 setuptools==30.0.0
 six>=1.9.0,<2
-thrift==0.9.1
+thrift>=0.9.1
```

Comment:
Given the backwards-compatibility store of python thrift (changing the wire-format of floats, for example... cc @patricklaw ), an open-ended dep seems like a bad idea. This there a reasonable bound for this? And if not, can you add a comment as to why not?